ot3_state_manager/ot3_state_manager/util.py

```diff
@@ -3,28 +3,92 @@
 from __future__ import annotations
 
 import enum
-import hashlib
+from enum import Enum, unique
 
+from opentrons.hardware_control.types import OT3Axis
 
-class Direction(enum.Enum):
+
+class Direction(int, enum.Enum):
     """Class representing direction to move axis in."""
 
-    POSITIVE = "+"
-    NEGATIVE = "-"
+    POSITIVE = 1
+    NEGATIVE = 0
+
+    @classmethod
+    def from_int(cls, direction_val: int) -> Direction:
+        """Parse integer into direction."""
+        if direction_val == Direction.NEGATIVE:
```

Comment:
can this be something more like 
```
if direction_val not in (Direction.NEGATIVE.value, Direction.POSITIVE.value):
    raise ValueError(...)
return Direction(direction_val)
```

Reply:
add3d48d0400a12a4af54bf5312c0140c5418fb1